How can I become a SDET?

peterintest opened the pull request.

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+today I work as a senior SDET, developing test automation frameworks and lead quality strategies. 
+Back in 2011, I knew very little about testing and software development and I wasn't even aware of testing as a career 
+option. However, my first job ignited a passion for learning and curiosity in computer science and testing where every 
+day was a school day, and I could learn something new. Passion is not something that just happens overnight, 
```
Love this! "every day is a school day"

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+it’s a process that takes time and sometimes obstacles get in the way, or you can get disheartened by not understanding 
+a concept at first; but the key is to persist with something a little longer even when the novelty has worn off, and 
+the challenges start. Find your passion as it will keep you dedicated, my advice would be to try many things and stick 
+with something that you are curious about and find engaging.
```
This is _such_ good advice for any new grads.

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+one of your customers. Find out what skills you bring to the table, start developing skills as soon as you can, don’t 
+fear perfection and just start and take small steps each day or week. One of the first skills I wanted to learn in 2011 
+was the ability to code in Java, and I started by finding opportunities to apply what I was learning at work and set 
+myself goals for studying so much each week. Talk to you manager, a colleague or a friend and see how people can support 
```
On line 24 I think `you` should be `your`

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+
+Develop a growth mindset, this is very important. Working in technology requires continuous learning, it’s one of the 
+reasons I like working in the industry as I get the opportunity to learn new technologies, approaches or methodologies 
+as part of my daily work. You must set yourself some goals and be accountable to working towards them, I write my goals 
```
Just a suggestion, you could add a semicolon here instead of a comma.

Current: `You must set yourself some goals and be accountable to working towards them, I write my goals`

Recommended: `You must set yourself some goals and be accountable to working towards them; I write my goals`

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+
+Nobody likes failure, especially when you describe an event in such a way, whether that be a failed project, business or 
+promotion. I tend to find I have learnt more from failure than I have from success, I see failure as a learning 
+opportunity so prefer to call it that.
```
Such a great outlook.

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+Whether you want to be a SDET, manager or software engineer find out what skills you lack or are weak in and create a 
+plan to work towards improving them. Ask for feedback on your work and skills and be prepared to actively listen to 
+other opinions, both positive and constructive feedback. Even when something seemingly went well, ask for feedback! 
+You can then form a plan on how to improve, for example I remember giving a presentation to an engineering team on a 
```
Another area a semicolon could be used rather than a comma:

Current: `You can then form a plan on how to improve, for example I remember giving a presentation to an engineering team on a `

Recommended: `You can then form a plan on how to improve; for example I remember giving a presentation to an engineering team on a `

kimbocote commented on peterjohnson/PETER_JOHNSON.md:
```diff
@@ -0,0 +1,77 @@
+If you find you are naturally enjoying one skill more than another, why not consider a role that allows you to use your 
+strongest skill the most. For example, if you enjoy communicating and leadership more than coding then maybe working as 
+a SDET may not be a good fit as it’s very engineering focused but working as a QA engineer allows you to utilise your 
+communication skills to influence the overall testing strategy for example, and you can end up in a QA lead position.
```
Consider adding an 'About the Author' section with your name as a title and where people can find more information about you. James Thomas has one in his PR.

kimbocote reviewed (changes requested):
Great story!  Most of my comments are stating things I liked.  There are two that request changes and two with recommendations.

Changes are on lines: 24 & 77

Recommendations are on lines: 35 & 49

melissafishersagecom reviewed (approved):
What a motivational story, peter - can't wait to share this with the world.

peterintest commented:
Thanks @kimbocote and @melissafishersagecom for the comments and review.

@kimbocote I've addressed your comments, feel free to review again and approve

kimbocote reviewed (approved):
Approved!
